lighthouse-core/audits/metrics/estimated-input-latency.js

```diff
@@ -12,7 +12,7 @@ const UIStrings = {
   /** The name of the metric that marks the estimated time between the page receiving input (a user clicking, tapping, or typing) and the page responding. Shown to users as the label for the numeric metric value. Ideally fits within a ~40 character limit. */
   title: 'Estimated Input Latency',
   /** Description of the Estimated Input Latency metric that estimates the amount of time, in milliseconds, that the app takes to respond to user input. This description is displayed within a tooltip when the user hovers on the metric name to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'The score above is an estimate of how long your app takes to respond to user ' +
```

Comment:
Ah yes good catch we should definitely change from `The score above` to `Estimated Input Latency` 👍 

I'm not sure I like `a rough guess` rather than `an estimate`, that feels like it doesn't give us *quite* enough credit for how it's computed :)

Reply:
Yeah, simply `estimate` is probably more meaningful. I want to avoid the word twice in a sentence. Maybe there's an alternative word around.

Reply:
Yeah I'm not sure about "rough guess" maybe "an approximation"?

Reply:
+1 on keeping "estimate"